### PR TITLE
Nav bar

### PR DESCRIPTION
**Pull request by daleinen7**
[ticket](https://trello.com/c/xorZwrUa/88-develop-navigation-bar)

### CODE REVIEW

File: _includes/navbar-w-links.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!-- NAV -->
+
+<nav class="navbar">
+	<button onClick="toggleMenu()" class="menu-icon"><img src="/images/hamburger-open-menu.svg" alt="open menu"></button>
```
Comment (jrapala) [inline _includes/navbar-w-links.html:4]: "Open Menu" remains the same whether it's a hamburger icon or an x icon
I also suggest we add an aria-label to the button instead and add aria-hidden to the img, and use aria-expanded to communicate if the menu is open or closed

**Review by jrapala — CHANGES_REQUESTED** [review]
Good job! Left a few comments, mostly some accessibility tips. There also seems to be a border missing on the mobile menu:

Screen Shot 2021-06-09 at 6 45 02 PM

File: scss/main.scss [hunk 2]
```diff
@@ -22,6 +22,11 @@ body {
   color: $black;
 }
 
+.prevent-scrolling {
```
[the remark this answers is not in the document]
Comment (daleinen7) [reply scss/main.scss]: Might be more of a designer question if my solution works or not but let me know what you think.

File: _includes/navbar-w-links.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+				<a href="https://github.com/Code-For-Chicago/" target="_blank">
+						<img class="social-img" src="/images/github.svg" alt="GitHub"/>
+				</a>
+			</div>
+		</div>
+		<a href="https://www.codeforamerica.org/donate-to-a-brigade?utm_campaign=Code%20for%20Chicago&utm_source=Brigade%20site" target="blank" class="donate-link">Donate</a>
+		<a href="https://www.codeforamerica.org/donate-to-a-brigade?utm_campaign=Code%20for%20Chicago&utm_source=Brigade%20site" target="blank" class="short-button donate-button">Donate</a>
+	</div>
+</nav>
+<nav class="mobile-nav"></nav>
```
[the remark this answers is not in the document]
Comment (daleinen7) [reply _includes/navbar-w-links.html]: I was a little unclear on the best solution so let me know what you think.

File: js/nav.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  })
+  mobileNav.firstChild.appendChild(extNav.cloneNode(true)).className = 'external-links-mobile'
+}
+initMobileNav();
+
+const toggleMenu = ()=> {
+  // toggle menu icon
+  menuIcon.parentNode.ariaLabel = 
+    (menuOpen)
+    ? 'open'
```
Comment (jrapala) [inline js/nav.js:22]: Let's do "open menu" and "close menu" instead. It's not clear it's a menu just yet.

File: _includes/navbar-w-links.html [hunk 5]
```diff
@@ -0,0 +1,52 @@
+<!-- NAV -->
+
+<nav class="navbar" aria-label="main">
+	<button aria-label="open" onClick="toggleMenu()" class="menu-icon"><img src="/images/hamburger-open-menu.svg" alt="open menu" aria-hidden="true"></button>
```
Comment (jrapala) [inline _includes/navbar-w-links.html:4]: Just an FYI, if there is an `aria-hidden`, the `alt` will not be read (so the `alt` isn't really necessary here)

**Review by jrapala — CHANGES_REQUESTED** [review]
Just two small a11y-related requests

**Review by jrapala — DISMISSED** [review]

**Pull request merged**